.github/workflows/pytest.yml

```diff
@@ -1,6 +1,6 @@
 name: ampl
 
-on: [push]
+on: [push, pull_request]
```

Comment:
Now we can get builds on PRs as well as commits directly to the repo. The main difference here:
1. `push` will run whenever the AMPL core team is developing (since they have write access)
2. `pull_request` will run if someone outside AMPL submits a PR